cypress/e2e/CMS/$SiteSections/index.js

```diff
@@ -0,0 +1,97 @@
+});
+
+And('user selects {string} from RELATIONS dropdown', (selectSiteSection) => {
+    cy.get('#edit-parent').should('be.visible').select(selectSiteSection);
+});
+
+And('user removes site section', () => {
+    cy.get('#edit-field-site-section-current-items-0-remove-button').click();
+});
+
+When('user selects {string} site section', (selectSiteSection) => {
+    cy.get("input[value='Select Site Section']").click({ force: true });
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find('input[name="computed_path_value"]').type(selectSiteSection);
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find('input#edit-submit-site-section-browser').click({ force: true });
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find(`td:contains('${selectSiteSection}')`).first().parent()
+        .find('td.views-field.views-field-entity-browser-select input').check();
+    cy.getIframeBody('iframe.entity-browser-modal-iframe')
+        .find("input[id='edit-submit'][value='Select Site Section']").click();
+});
+
+Given('user is navigating to the front end site with path {string}', (purl) => {
+    cy.on('uncaught:exception', (err, runnable) => {
+        if (err.message.includes('Failed to load script https://assets.adobedtm.com')) {
+            return false;
+        }
+        return true;
+    })
+    cy.visit(`${frontEndBaseUrl}${purl}`, { retryOnStatusCodeFailure: true });
+});
+
+Then('the current page is {string}', (title) => {
+    cy.get(`li.usa-sidenav__item a:contains("${title}")`).should('have.attr', 'aria-current', 'page');
+});
+
+And('the following nav children are displayed', (dataTable) => {
+    for (const { label } of dataTable.hashes()) {
+        cy.get('a[class="usa-current"]').siblings('ul').find(`a:contains("${label}")`).should('be.visible')
+    }
+});
+
+And('{string} has been selected', (titleText) => {
+    cy.get(`div[id*='edit-field-site-section-current-items-0']`).should('be.visible').and('contain.text', titleText);
+});
+
+And('{int} level up section is {string}', (level, label) => {
+    let ancestors;
+    cy.document().then(doc => {
+        ancestors = doc.querySelectorAll('.usa-current.usa-current--nci-ancestor').length;
+        cy.get('.usa-current.usa-current--nci-ancestor').eq(ancestors - level).should('have.text', label);
+    })
+
+});
+
+And('user selects a checkbox next to title {string} with url {string} from the list of content', (title, lastPartUrl) => {
+    cy.get(`a:contains("${title}")`).parent().parent().find('input.form-checkbox').check();
+});
+
+And('user clicks on the {string} action button', (buttonLabel) => {
+    cy.get(`input[value='${buttonLabel}']`).first().click();
+});
+
+And('user selects {string} link under {string}', (children, parentName) => {
+    cy.get(`a.menu-item__link:contains("${parentName}")`).parent().parent().find(`a:contains("${children}")`).click();
```

Comment:
step at line 89 and line 93 has the same code - when the locators and actions are duplicated for 2 different steps, those steps should be combined into one generic

Reply:
cypress test run after updating the step:

![image](https://user-images.githubusercontent.com/28455973/213756147-7e457bdb-84cf-47ee-857b-7f68380fd7a8.png)